qiskit_experiments/library/characterization/zz_ramsey.py

```diff
@@ -0,0 +1,339 @@
+        # Override the default of get_processor() which is "1" * num_qubits. We
+        # only fit the probability of the target qubit.
+        self.analysis.set_options(outcome="1")
+        self.set_experiment_options(**experiment_options)
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default values for the :math:`ZZ` Ramsey experiment.
+
+        Experiment Options:
+            delays (list[float]): The list of delays that will be scanned in
+                the experiment, in seconds. If not set, then ``num_delays``
+                evenly spaced delays between ``min_delay`` and ``max_delay``
+                are used. If ``delays`` is set, ``max_delay``, ``min_delay``,
+                and ``num_delays`` are ignored.
+            max_delay (float): Maximum delay time to use.
+            min_delay (float): Minimum delay time to use.
+            num_delays (int): Number of circuits to use per control state
+                preparation.
+            zz_rotations (float): The "fake" rotation added to qubit 0 uses a
+                frequency that gives this many rotations in the case where
+                :math:`f_{ZZ}` is 0.
+        """
+        options = super()._default_experiment_options()
+        options.delays = None
+        options.min_delay = 0e-6
+        options.max_delay = 10e-6
+        options.num_delays = 50
+        options.zz_rotations = 5
+
+        return options
+
+    @property
+    def delays(self):
+        """Delay values to use in circuits"""
+        # This method allows delays to be set by the user as an explicit
+        # sequence or as a minimum, maximum and number of values for a linearly
+        # spaced sequence.
+        options = self.experiment_options
+        if options.delays is not None:
+            return options.delays
+
+        return np.linspace(options.min_delay, options.max_delay, options.num_delays)
+
+    def _parameterized_circuits(
+        self, delay_unit: str = "s"
+    ) -> Tuple[QuantumCircuit, QuantumCircuit]:
+        """Circuits for series 0 and 1 with several parameters
+
+        Circuits are generated with parameters for:
+
+            pi: the mathematical constant
+            f: the synthetic frequency added onto the circuit
+            dt: the conversion factor from the units used in the circuits'
+                delay instructions to time in seconds.
+            delay: the length of each delay instruction in each circuit
+
+        The circuits are parameterized with these parameters so that
+        ``QuantumCircuit.draw()`` produces a nice representation of the
+        parameters in the circuit.
+
+        Args:
+            delay_unit: the unit of circuit delay instructions.
+
+        Returns:
+            Circuits for series 0 and 1
+        """
+        metadata = {
+            "experiment_type": self._type,
+            "qubits": self.physical_qubits,
```

Comment:
I don't think these values are necessary because they are also stored in the experiment data and not used for validation when loading an experiment with jobs. This just increases the payload size.

```suggestion
            "experiment_type": self._type,
            "qubits": self.physical_qubits,
```

Reply:
I copied these from RamseyXY. I think they are left over from the original design of having the experiment and analysis completely independent so any metadata needed to be attached to circuits. I think now we allow experiment-wide metadata? I haven't looked at this closely. I removed the entries in c787899989e93d0ad9bf50cf5450dc0e9926d5e1.